### PR TITLE
Add required classes and css for Labs styling on PhotoEssays

### PR DESCRIPTION
**Pull request by katebee**
## What does this change?

- Adds CSS to the photo essay to target the `content__section-label--advertisement` to apply the same styling as found in `_article-immersive.scss` ([link](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/module/content/_article-immersive.scss#L251))

- Labs headlines always use the sans-serif styling

- Labs section labels are always large, teal and sans-serif 

- Removes an unused class from the templates (see comments)

picture 901

## What is the value of this and can you measure success?

**PhotoEssays** are a subset of **Immersives**, however, they use a separate template that does not apply the same classes is the article `isPaidForContent`. This PR applies the missing classes and styles to the template. [Here is an example](https://www.theguardian.com/cricket-has-no-boundaries/2017/jun/20/wicket-maidens-the-surprising-history-of-womens-cricket) immersive where the styling is correct.

- Labs content looks correct and consistent across different article types

- No delays to publication of scheduled content

## Does this affect other platforms - Amp, Apps, etc?

Nope

#### STYLES BEFORE:

picture 891

#### STYLES AFTER:

picture 890

##### CLASSES BEFORE: 

picture 903

##### CLASSES AFTER: 

picture 902

### CODE REVIEW

File: common/app/views/fragments/photoEssay.scala.html [hunk 1]
```diff
@@ -8,15 +8,17 @@
...
+)) { case (hasEmbed, hasMainMedia, hasVideo, isPaidContent) =>
   <style>
     @Html(common.Assets.css.inlinePhotoEssay)
   </style>
 
     <div class="content--photo-essay @RenderClasses(Map(
             "immersive-main-media" -> hasMainMedia,
-            "photo-essay--video" -> hasVideo
+            "photo-essay--video" -> hasVideo,
+            "immersive-main-media__advertisment" -> isPaidContent
```
Comment (kelvin-chappell) [inline common/app/views/fragments/photoEssay.scala.html]: Is this a typo or is it the class name?

**Review by kelvin-chappell — COMMENTED** [review]
👍

**Review by rich-nguyen — APPROVED** [review]

File: common/app/views/fragments/photoEssay.scala.html (hunk 1 above)
Comment (katebee) [inline common/app/views/fragments/photoEssay.scala.html]: Eagle eyes! Hah, that class is only used in two places, the outmost \<DIV\> of the templates in`immersiveMainMedia.scal.html` and now `photoEssay.scala.html`. 

This class isn't currently linked to any CSS... or doing anything at all. Neither is the correct spelling of `immersive-main-media__advertisement`. 😅 Which leads me to the question, do I fix the spelling and keep applying the classes? or do I just delete this line and line 23 of `immersiveMainMedia.scal.html`?

**Comment by katebee** [comment]
After a bit of digging I am fairly confident that neither `immersive-main-media__advertisement` or `immersive-main-media__advertisment` are needed, so I'm going to delete these classes from the templates.

picture 901

🔥 🔥 🔥

File: common/app/views/fragments/photoEssay.scala.html (hunk 1 above)
Comment (rich-nguyen) [inline common/app/views/fragments/photoEssay.scala.html]: Oh, I thought the class was affecting the css styling?

**Pull request merged**